src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3407,6 +3407,19 @@ void SpellMgr::LoadSpellInfoCorrections()
                 break;
             // ENDOF ISLE OF CONQUEST SPELLS
             //
+            // FIRELANDS SPELLS
+            //
+            case 99253: // Torment Searcher
+                spellInfo->RangeEntry = sSpellRangeStore.LookupEntry(EFFECT_RADIUS_15_YARDS);
```

Comment:
This doesn't work at all. You are mixing range (distance between caster and spell destination, defined per spell) with radius (distance between spell destination and target, defined per effect)